app/internal/notify/client.go

```diff
@@ -118,6 +121,12 @@ func (c *Client) TemplateID(id TemplateId) string {
 			return "1c4d5b24-fc7d-45ee-be40-f1ccda96f101"
 		case CertificateProviderPaperMeetingPromptSMS:
 			return "6be11b4a-79f9-441e-8afe-adff96f7e7fc"
+		case CertificateProviderDigitalLpaDetailsChangedNotSeenLpaSMS:
+			return "19948d7d-a2df-4e85-930b-5d800978f41f"
+		case CertificateProviderPaperLpaDetailsChangedSMS:
+			return "d363a56f-e802-4f88-bd09-80b8c9e9d650"
+		case CertificateProviderDigitalLpaDetailsChangedSeenLpaSMS:
+			return "71d21daa-11f9-4a2a-9ae2-bb5c2247bfb7"
```

Comment:
this is getting to be a big list, maybe next time we should think about changing it, somehow

Reply:
Agree 👍